perl_modules/Atlas/ZoomaClient.pm

```diff
@@ -191,22 +192,28 @@ Given a property type and value, return a hash containing the query results.
 
 sub _query_zooma {
 
-    my ( $self, $propertyType, $propertyValue ) = @_;
+    my ( $self, $propertyType, $propertyValue, $organism ) = @_;
     
     my ( $propertyType4url, $propertyValue4url ) = ( $propertyType, $propertyValue );
 
     # First we need to make sure the property type and value are in the right
     # format for a URL.
     $_ = url_encode_utf8( $_ ) for ( $propertyType4url, $propertyValue4url );
+
+    my $ontology_lookup = zooma_ontology_lookup ("zooma_ontologies.tsv");
```

Comment:
I suspect that we could also bring this some levels up and then avoid opening this file on every single query.